ui/src/views/404/index.jsx

```diff
@@ -1,72 +1,86 @@
-// @flow
+// @follow
```

Comment:
What's `@follow`?

Reply:
It's the next version of `flow`

Or maybe a mistaken transcription 😅

Reply:
k cool, was trying to Google that 😆